client/scss/components/_blockquote.scss

```diff
@@ -0,0 +1,54 @@
+.blockquote {
+  @include font('HNM2');
+  text-align: center;
+
+  &--wb {
```

Comment:
Composing of SASS in this way makes it really hard for static analysis later.
Unless there's a tools that does SASS static analysis.